### PR TITLE
(RFR) Add reviews

### PR DESCRIPTION
**Pull request by N8Goodman**

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class ReviewsController < ApplicationController
+  def create
+    @bookstore = Bookstore.find(params[:bookstore_id])
+    @review = Review.new(review_params)
+    @review.bookstore = @bookstore
```
Comment (mlg-) [inline app/controllers/reviews_controller.rb]: A `merge` in the `review_params` would shorten up this controller nicely.

File: app/controllers/reviews_controller.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  def edit
+  end
+
+  def update
+  end
+
+  def destroy
+  end
+
+  private
```
Comment (mlg-) [inline app/controllers/reviews_controller.rb:31]: Fix this.

File: app/views/bookstores/show.html.erb [hunk 3]
```diff
@@ -25,4 +25,20 @@
...
+          <div class="review_body">
+            <%= review.body%>
+            <%= review.overall_rating %><%= review.price_rating %><%= review.customer_service_rating %>
+          </div>
+        </li>
+      <% end %>
+    </ul>
+  <br>
+
+  <%= render './reviews/form' %>
```
Comment (mlg-) [inline app/views/bookstores/show.html.erb]: You shouldn't need the dot before the path...were you having issues without it?

File: app/views/reviews/_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,48 @@
+<h4>Add Review:</h4>
+<br>
```
Comment (mlg-) [inline app/views/reviews/_form.html.erb]: Use CSS to create space between elements, not `<br>` tags.

File: app/views/reviews/_form.html.erb [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+  <div class="required_field">
+    <%= f.label :body, "Comments:" %>
+    <%= f.text_area :body %>
+  </div>
+
+  <div>
+    <%= f.submit "Add Review" %>
+  </div>
+
```
Comment (mlg-) [inline app/views/reviews/_form.html.erb]: Omit this blank line.

File: spec/features/user_adds_a_review_spec.rb [hunk 6]
```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+feature "user sees a specific bookstore page" do
+  zip = "18900"
```
Comment (mlg-) [inline spec/features/user_adds_a_review_spec.rb]: Hmm, why are you doing this? Also, why not use a factory? I presume you'll have to make lots of these going forward!

File: spec/features/user_adds_a_review_spec.rb [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+      url: "http://www.bookendswinchester.com/",
+      img_url: "http://www.booksite.com/cilla/bookends/CAIZ8923.jpg",
+      description: "A nice little bookstore",
+      beverage: false,
+      food: true
+    )
+  end
+
+  scenario 'user visits page for a specific bookstore' do
+    visit '/'
```
Comment (mlg-) [inline spec/features/user_adds_a_review_spec.rb]: Use path helpers. `visit root_path`

File: spec/features/user_adds_a_review_spec.rb [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+    click_link 'Bookends'
+    fill_in "Comments:", with: "This place is awesome!"
```
Comment (mlg-) [inline spec/features/user_adds_a_review_spec.rb:35]: This test is not really testing the existence of a specific page, it's testing adding a comment. Change the scenario & feature statements above.

File: spec/features/user_adds_a_review_spec.rb [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+    select '5', from: 'Overall Rating'
+    select '5', from: 'Price Rating'
+    select '5', from: 'Variety Rating'
+    select '5', from: 'Customer Service Rating'
+
+    click_button 'Add Review'
```
Comment (mlg-) [inline spec/features/user_adds_a_review_spec.rb:42]: This test is not complete! There is no assertion (an expect statement). Modify it to actually test that the review was added successfully. Include a test case for an error state.

File: app/controllers/reviews_controller.rb [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+
+    def review_params
```
Comment (alxjrvs) [inline app/controllers/reviews_controller.rb]: This should not be Indented.

**Pull request merged**